README.md

```diff
@@ -60,15 +60,13 @@ Also, in `onisite/urls.py`, add the Nebraska about URLs:
 
 ```
 from django.conf.urls import url, include
+from onisite.plugins.featured_content import views
 
 urlpatterns = [
-  url('', include("onisite.plugins.calendar.urls")),
-  url('', include("onisite.plugins.featured_content.urls")),
+  url(r'^calendar-', include("onisite.plugins.calendar.urls")),
+  url(r'^$', views.featured, name="featured_home"),
```

Comment:
can you explain how this is working?  I don't actually see anything in the views file related to the featured pages, but maybe it's finding it globally (therefore including the plugins)?  I assume that by specifying, we would need to change this line in order to use the featured pages API path, right?

Reply:
The line above imports the `views` object from the plugin